build/build.sh

```diff
@@ -57,6 +57,16 @@ ${GO_BUILD_CMD} \
   -buildvcs=false \
   -o "${TARGETS_DIR}/nginx-ingress-controller" "${PKG}/cmd/nginx"
 
+echo "Building ${PKG}/cmd/nginx with Go's symbol table"
```

Comment:
whats the difference between `scannable-nginx-ingress-controller` and `nginx-ingress-controller` build options?